### PR TITLE
:wrench: Implement flowchart connectors

### PR DESCRIPTION
**Pull request by OGENDI**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes # (issue) - *Example*: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Screenshot (optional)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by pmutua** [comment]
@Derick Ogendi great! you've make a PR please edit it this is only the PR template it doesn't have information about what you did and what issue this is related to also do you have a screen shot showing that your implementation works?

File: libs/features/convs-mgr/stories/blocks/library/block-options/src/lib/providers/jsplumb-decorator.function.ts [hunk 1]
```diff
@@ -17,10 +18,13 @@ export function _JsPlumbComponentDecorator(sourceElement: Element, jsPlumb: Brow
     // Where to position the anchor
     anchor: "Right",
     connector: {
-      type: BezierConnector.type,
+      // type: BezierConnector.type,
+        type: FlowchartConnector.type,
```
Comment (pmutua) [inline libs/features/convs-mgr/stories/blocks/library/block-options/src/lib/providers/jsplumb-decorator.function.ts:22]: @OGENDI leave this as it was the connector type should be of type `BezierConnector.type`.

**Review by pmutua — CHANGES_REQUESTED** [review]

File: libs/features/convs-mgr/stories/blocks/library/block-options/src/lib/providers/jsplumb-decorator.function.ts (hunk 1 above)
Comment (jrosseel) [reply libs/features/convs-mgr/stories/blocks/library/block-options/src/lib/providers/jsplumb-decorator.function.ts:22]: @pmutua , it is actually correct. The Flowchart should be used.

@OGENDI , which issue number is this PR for please?

**Comment by jrosseel** [comment]
Closed due to inactivity

**Pull request closed without merging**